packages/office-ui-fabric-react/src/components/Rating/Rating.test.tsx

```diff
@@ -23,28 +23,31 @@ describe('Rating', () => {
       threwException = true;
     }
     expect(threwException).toEqual(false);
-
     let renderedDOM = ReactDOM.findDOMNode(rating as React.ReactInstance);
 
-    let ratingInputs = renderedDOM.querySelectorAll('.ms-Rating-input');
+    let ratingButtons = renderedDOM.querySelectorAll('.ms-Rating-button');
+    let ratingFrontStars = renderedDOM.querySelectorAll('.ms-RatingStar-front');
+
+    const checkState = (ratingToCheck: number, state: string) => {
+      let iconElement = ratingFrontStars[ratingToCheck - 1] as HTMLElement;
+      let width = iconElement.style.width;
+      expect(width).to.be.eq(state, 'Rating star width should be ${state}');
```

Comment:
this is wrong. If you `npm install` latest and open this file, you should see red squiggles in vscode. That's because we changed to Jest from mocha/chai.

this should be

```
expect(width).toEqual(state);
```

See other red squigglies as well in this file. That should fix your build.

Reply:
Updated